src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.cpp

```diff
@@ -204,6 +204,28 @@ tnsr::aa<DataType, SpatialDim, Frame> compute_pi(
   return pi;
 }
 
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::a<DataType, SpatialDim, Frame> compute_spacetime_normal_one_form(
+    const Scalar<DataType>& lapse) noexcept {
+  tnsr::a<DataType, SpatialDim, Frame> normal_one_form{
```

Comment:
This is not what we had discussed, you should make the Tensor with `make_with_value`, not the `DataType`